envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -222,6 +222,17 @@ message HttpConnectionManager {
   // downstream. If this is false or not set, Envoy will instead strip the
   // "Expect: 100-continue" header, and send a "100 Continue" response itself.
   bool proxy_100_continue = 18;
+
+  // If
+  // :ref:`use_remote_address
+  // <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.use_remote_address>`
+  // is true and remote_address_ipv4_mapped is true and the remote address is an IPv4 address, the
+  // address will be mapped to IPv6 before it is appended. This is useful for testing compatibility
```

Comment:
Can you finish the sentence here "appended" with what it is appended to (basically a short summary of the above docs). Otherwise it's unclear what the action is to the casual reader.